src/DynamoDbQueryBuilder.php

```diff
@@ -547,9 +541,8 @@ protected function getAll($columns = [], $limit = -1, $use_iterator = true)
 
         foreach ($iterator as $item) {
             $item = $this->model->unmarshalItem($item);
-            $model = $this->model->newInstance($item, true);
-            $model->setUnfillableAttributes($item);
-            $model->syncOriginal();
+            $model = $this->model->newInstance([], $exists = true);
+            $model->setRawAttributes($item, $sync = true);
```

Comment:
Just to avoid extra variables that we don't need, could you remove `$exists` and `$sync`:

```php
$model = $this->model->newInstance([], true);
$model->setRawAttributes($item, true);
```